src/scripts/main.js

```diff
@@ -1,3 +1,19 @@
 'use strict';
 
-// write code here
+const allLi = document.querySelectorAll('li');
+
+for (const item of allLi) {
+  if (item.querySelector('ul')) {
+    const span = document.createElement('span');
+
+    span.className = 'headerName';
```

Comment:
you've added it only to check in the listener?
instead of that,  you can check if it's "SPAN" tag (FYI)